ext/ffi_c/FunctionInfo.c

```diff
@@ -110,6 +113,23 @@ fntype_free(void *data)
     xfree(fnInfo);
 }
 
+static size_t
+fntype_memsize(const void *data)
```

Comment:
I think I accounted for all held data, but worst case these function don't have to be 100% accurate.

Reply:
A FunctionInfo can have a closurePool associated for all closures that are created in relation to this FunctionInfo. I'll add at least the management data of the ClosurePool to the size.

Reply:
Sounds good, I'll have a look at it.

We canalso report the size of the pool divided by the number of refs to it, I've done that in `msgpack-ruby`.